### ISSUE TITLE
Input with name="identifier" gets unfocused.

### ISSUE THREAD

**Issue by 101budspencer** [emilkowalski/vaul#170]
If I put an input in the drawer with name="identifier" for example:
'<input type="text" name="identifier" />' and then click the input, the body gets focused.

So on click the input blurs and focus is on the html body.

If I remove the name the input can be focused.

But I need the name to be "identifier".

**Comment by 101budspencer** [comment]
![Screenshot_2](https://github.com/emilkowalski/vaul/assets/52600520/d758f973-537a-4b9e-a549-0acab6785360)
![Screenshot_3](https://github.com/emilkowalski/vaul/assets/52600520/584d1a82-806f-4607-bec4-b4d6fda10721)

This happened everytime I click on the input.

**Comment by 101budspencer** [comment]
@emilkowalski is this a general radix ui issue?

**Comment by michaelmikh** [comment]
I guess this might be somewhat related: https://github.com/emilkowalski/vaul/issues/128

**Comment by 101budspencer** [comment]
@michaelmikh so how do I solve it?

**Comment by 101budspencer** [comment]
https://github.com/emilkowalski/vaul/assets/52600520/2b5e89a7-cc6b-4e0d-8b5f-f3fd633b1c00

@emilkowalski @michaelmikh I think this might help

Again I inspected the current focused element and the issue is that something causes the body to focus on click on the input.

**Comment by michaelmikh** [comment]
> @michaelmikh so how do I solve it?

I don't have a particular solution for this, sorry. You either have to look into that yourself or wait for 'vaul' maintainer to address it in some way.

**Comment by 101budspencer** [comment]
So you just stopped using the drawer?

Am Sa., 25. Nov. 2023 um 18:21 Uhr schrieb Michael ***@***.***

**Comment by michaelmikh** [comment]
No, I still use it, but as for now I don't do inputs or stuff inside.
